### ISSUE TITLE
Error page when double-clicking the login button 

### ISSUE THREAD

**Issue by a5e** [auth0/lock#1979]
### Describe the problem

When double-clicking the login button, the user is redirected to something.auth0.com/login/callback with the error message:

>Oops!, something went wrong
>
>There could be a misconfiguration in the system or a service outage. We track these errors automatically, but if the problem persists feel free to contact us.
>Please try again.

>access_denied: Password login via OIDC-conformant clients with externally-hosted login pages is unsupported. Alternatively, login could have been initiated from the wrong place (e.g., a bookmark).

But the login attempt was successful and accessing the main url for our app shows that the user is correctly authenticated.

### What was the expected behavior?

The login should perform without any error.

### Reproduction

- Step 1.. Enter correct credentials
- Step 2.. Double click the login button
- Step 3.. You should see a similar page 
![error](https://user-images.githubusercontent.com/1247692/113562492-7a414480-9606-11eb-83a3-0547eb5d4881.jpg)

### Environment

- **Version of this library used:** 11.26
- **Which framework are you using, if applicable:** None
- **Other modules/plugins/libraries that might be involved:** None
- **Any other relevant information you think would be useful:** Could double-clicking be prevented by auth0/lock itself?

**Comment by a5e** [comment]
It turns out another dev removed a class from the button after it was displayed, hence why it was still possible to double-click the sign-in button. I'm sorry I made you test this, I did not expected this and should not have open this issue in the first place.

**Comment by Nadeembhat** [comment]
i am having the same issue 
Android Google Pixel- Emulator API 29
Google Chrome Browser
Steps to reproduce is same as above mentioned by @a5e

**Comment by stevehobbsdev** [comment]
@Nadeembhat In the case of @a5e, their issue was do with removing a class after the page was rendered, are you in a similar situtation where something is modifying the button rendered by Lock?

**Comment by Nadeembhat** [comment]
@stevehobbsdev let me gather more information about this.
I am not using lock in android, and loginbutton is clickable more than once.if you can help me how can i lock the UI element for login button so that i can click only once .
Thanks

**Comment by stevehobbsdev** [comment]
@Nadeembhat  once you have done your own research into your problem, please raise a new issue rather than adding to this one so that it doesn't get lost 👍

**Comment by Nadeembhat** [comment]
@stevehobbsdev  Sure thanks for the quick response

**Comment by alex-restless** [comment]
We've just had the same problem so will be applying the ability to click only once too.
